server/game/game.js

```diff
@@ -343,9 +343,8 @@ class Game extends EventEmitter {
             case 'active plot':
                 this.callCardMenuCommand(player.activePlot, player, menuItem);
                 break;
+            //agenda and play area can behave the same now as the alliance agenda allows you to have more than one agenda card that are clickable
```

Comment:
Ah, I didn't notice the new Agenda was also a banner. We might want to adjust the UI in some way, but this should be fine for now.

Reply:
Created #3072 for this